bot/engine/src/main/kotlin/engine/action/ActionMetadata.kt

```diff
@@ -28,5 +28,6 @@ data class ActionMetadata(
         /** Message which is a reply to another. */
         var replyMessage: ActionReply = ActionReply.UNKNOWN,
         /** Message which contains a quote. */
-        var quoteMessage: ActionQuote = ActionQuote.UNKNOWN
+        var quoteMessage: ActionQuote = ActionQuote.UNKNOWN,
+        var lockOrchestration: Boolean = false
```

Comment:
With this attribute we introduce the notion of orchestration in the TOCK core.
Tock should provide a mechanism allowing a third party module to enrich the metadata.